src/core/layout/grid/grid.js

```diff
@@ -66,6 +66,10 @@ export class GelGrid extends Phaser.GameObjects.Container {
         });
     }
 
+    getCurrentSelection() {
+        return this.choices()[this.page];
```

Comment:
I hate this but it's just how we are doing it!

Reply:
How would you want to do it instead?

Reply:
A getter?

Reply:
I just don't like `this.page` === selection. I wouldn't change it yet as we don't really require any more to be done in this area - and this works!